packages/alfa-rules/src/sia-r11/rule.ts

```diff
@@ -41,14 +34,13 @@ export const SIA_R11: Atomic.Rule<Device | Document, Element> = {
     return {
       applicability: () => {
         return map(
-          filter(querySelectorAll(document, document, isElement), element => {
-            return map(isLink(element, document, device), isLink => {
-              if (!isLink) {
-                return false;
-              }
-
-              return isExposed(element, document, device);
-            });
+          filter(querySelectorAll(document, document, isElement()), element => {
+            return isElement(builder =>
+              builder
+                .withNamespace(document, Namespace.HTML)
+                .withRole(device, document, Roles.Link)
+                .and(element => isExposed(element, document, device))
```

Comment:
`isExposed` could obviously be added as a predicate builder method. Maybe another time…